pkgs/os-specific/linux/dcgm/default.nix

```diff
@@ -104,36 +104,30 @@ in gcc11Stdenv.mkDerivation rec {
 
   hardeningDisable = [ "all" ];
 
+  strictDeps = true;
+
   nativeBuildInputs = [
-    addOpenGLRunpath
+    # autoAddOpenGLRunpathHook does not actually depend on or incur any dependency
+    # of cudaPackages. It merely adds an impure, non-Nix PATH to the RPATHs of
+    # executables that need to use cuda at runtime.
+    cudaPackages_12.autoAddOpenGLRunpathHook
```

Comment:
Why hardcode the vetsion?

Reply:
This package depends on 3 different versions of cudaPackages, none of which are the default `cudaPackages`. Instead of pulling in a *4th* variant of cudaPackages, just for this, I randomly chose one of the existing ones. It's a shame this hook is attached to versions like this.